esp/src/src/nls/es/hpcc.ts

```diff
@@ -2,6 +2,7 @@ export = {
     Abort: "Aborte",
     AbortedBy: "Abortado por",
     AbortedTime: "Abortado a las",
+    AbortSelectedWorkunits: "¿Cancelar unidad(es) de trabajo seleccionada(s)? Su ID de inicio de sesión se registrará para esta acción dentro de la(s) WU(s).",
```

Comment:
Perhaps a little picky but workunits was translated as unidades de trabajo, but at the end of message we the acronym WU